node/node.go

```diff
@@ -995,6 +1002,8 @@ func (n *Node) runManager(ctx context.Context, securityConfig *ca.SecurityConfig
 		PluginGetter:     n.config.PluginGetter,
 		RootCAPaths:      rootPaths,
 		FIPS:             n.config.FIPS,
+		DefaultAddrPool:  n.config.DefaultAddrPool,
```

Comment:
(on the fence if we should drop `Default..` from the name (i.e. it's the `AddresPool` that was configured; "default" may be a bit redundant)

Reply:
Code readability will be easy if we carry the defaultAddress pool name. We will be passing the information all the way libnetwork. so trying to keep same name. 
Whatever addrpool configured will be used as default address pool. Hence used same name